partial-update-tests/existing/swagger/README.md

```diff
@@ -0,0 +1,18 @@
+## Generate code
+```yaml
+input-file: ./specification/bodystring_updated.json
+java: true
+output-folder: ../../generated
+required-parameter-client-methods: true
+generate-client-as-impl: true
+namespace: fixtures.bodystring
+generate-client-interfaces: false
+sync-methods: all
+generate-sync-async-clients: true
+license-header: MICROSOFT_MIT_SMALL
+add-context-parameter: true
+models-subpackage: implementation.models
+context-client-method-parameter: true
+regenerate-pom: false
```

Comment:
these seems not needed, exception `namespace`

Reply:
updated